ga4gh/datarepo.py

```diff
@@ -385,6 +388,11 @@ def _readSystemTable(self, cursor):
         row = cursor.fetchone()
         self._repositoryVersion = config["repositoryVersion"]
```

Comment:
I was thinking about this, perhaps it should be ``schemaVersion``? Repository version seems to imply that we're somehow tracking all the data that's going in and out. What do you think?